healthcareai/tests/test_dev_supervised_model_class.py

```diff
@@ -24,75 +25,67 @@ def setUp(self):
         self.o.data_preparation(impute=True)
         self.o.random_forest(cores=1)
 
-    def runTest(self):
-
         self.assertAlmostEqual(np.round(self.o.au_roc, 6), 0.965070)
 
-    def tearDown(self):
-        del self.o
-
 
 class TestRFDevTuneTrueRegular(unittest.TestCase):
-    def setUp(self):
-        df = pd.read_csv(fixture('HCPyDiabetesClinical.csv'),
-                         na_values=['None'])
+    def test_random_forest_dev_tune_true_succeeds(self):
+        df = pd.read_csv(fixture('HCPyDiabetesClinical.csv'), na_values=['None'])
 
         # Drop uninformative columns
         df.drop(['PatientID', 'InTestWindowFLG'], axis=1, inplace=True)
 
         np.random.seed(42)
-        self.o = DevelopSupervisedModel(dataframe=df, model_type='classification',
-                                        predicted_column='ThirtyDayReadmitFLG')
-        self.o.data_preparation(impute=True)
+        o = DevelopSupervisedModel(dataframe=df, model_type='classification', predicted_column='ThirtyDayReadmitFLG')
+        o.data_preparation(impute=True)
```

Comment:
This is a little misleading because the refactor isn't complete - if you look at the `Example3_simple.py` that will show you what the API for the simple user will look like. The data_prep is a convenience pipeline for advanced users that can be used or overridden.